src/utils/helpers.js

```diff
@@ -0,0 +1,14 @@
+export function setButtonText(
+  btn,
+  isLoading,
+  defaultText = 'Save',
+  loadingText = 'Saving...',
+) {
+  if (isLoading) {
+    btn.setAttribute('disabled', 'true')
```

Comment:
`    btn.setAttribute('disabled', 'true')
`

Please, delete the disabling logic from `setButtonText` because it will cause problems with disabling the button in `.then`